frontend/src/components/common/modals/ConfirmModal/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { PropsWithChildren } from 'react';
+
+import { ButtonType } from '@/types/styles';
+
+import Button from '../../Button';
+import ModalBackground from '../ModalBackground';
+import ModalPortal from '../ModalPortal';
+
+import * as S from './styles';
+
+interface ConfirmModalButton {
+  type: ButtonType;
+  text: string;
+  handleClick: (e: React.MouseEvent) => void;
+}
+
+interface ConfirmModalProps {
+  confirmButton: ConfirmModalButton;
+  cancelButton: ConfirmModalButton;
+}
+
+const ConfirmModal: React.FC<PropsWithChildren<ConfirmModalProps>> = ({ children, confirmButton, cancelButton }) => {
```

Comment:
```suggestion
const ConfirmModal: React.FC<React.PropsWithChildren<ConfirmModalProps>> = ({ children, confirmButton, cancelButton }) => {
```

`React.PropsWithChildren`을 사용하면 불필요한 import를 줄일 수 있을 것 같아요!